index.html

```diff
@@ -635,9 +635,21 @@ <h3>
               </li>
             </ol>
           </li>
+          <li>If the port is [=identifiable=], perform the following steps:
+              <ol>
+                <li>Set |info|["{{SerialPortInfo/id}}"] to the [=identifier=]
+                  of the port.
+                </li>
+              </ol>
+            </li>
           <li>Return |info|.
           </li>
         </ol>
+        <p>
+          A serial port is <dfn>identifiable</dfn> if the port can be uniquely
+          identified through an <dfn>identifier</dfn> across the user agent
```

Comment:
There should be a note here similar to the one in Web Bluetooth on this not being a cross-site tracking ID: https://webbluetoothcg.github.io/web-bluetooth/#ref-for-dom-allowedbluetoothdevice-deviceid%E2%91%A1